DependencyInjection/DoctrineMigrationsExtension.php

```diff
@@ -98,6 +108,23 @@ public function load(array $configs, ContainerBuilder $container) : void
         $container->setParameter('doctrine.migrations.preferred_connection', $config['connection']);
     }
 
+    private function checkIfBundleRelativePath(string $path, ContainerBuilder $container) : string
```

Comment:
this code is copied from the current path resolutions used by jms-serializer, should be already production-tested